src/wasm-binary.h

```diff
@@ -331,10 +331,7 @@ enum Section {
   Event = 13
 };
 
-enum SegmentFlag {
-  IsPassive = 0x01,
-  HasMemIndex = 0x02,
-};
+enum SegmentFlag { IsPassive = 0x01, HasMemIndex = 0x02, Is64 = 0x04 };
```

Comment:
Hah, this was added to the wrong enum (should have been in `MemoryFlags`) but because the code uses the namespace and not the enum type, it worked anyway :)